### DISCUSSION TITLE
Image search

### DISCUSSION THREAD

**Discussion by nam157** [milvus-io/milvus#24725] [Q&A and General discussion]
how to use milvus for face recognition, how to build indexing data and search indexing data. Please, can you help me ?

**Comment by yhmo** [comment]
Milvus is a data base to help store and search vectors/embeddings, but it doesn't generate vectors/embeddings. To build a face recognition app, you will need other tools to recognize and convert human faces to vectors/embeddings. Then insert the vectors/embeddings into a Milvus server to search.

You can start from a bootcamp:
https://milvus.io/bootcamp
An images search example you might interest: https://github.com/towhee-io/examples/tree/main/image/reverse_image_search
